test/test_ripper_tags.rb

```diff
@@ -316,6 +316,55 @@ class C
     assert_equal '3: scope C.red',  inspect(tags[2])
   end
 
+  def test_extract_delegate
+    tags = extract(<<-EOC)
+      class C
+        delegate :foo,
+                 :bar, to: :thingy
+        delegate :x, to: :thingy, prefix: true
```

Comment:
Can `delegate` also work with instance variables like `def_delegator`, e.g. `delegate :x, to: :@thingy, prefix: true`? If so, could you add a test for that?